src/main/flight/pid.c

```diff
@@ -245,7 +245,7 @@ void pgResetFn_pidProfiles(pidProfile_t *pidProfiles)
 
 // Scale factors to make best use of range with D_LPF debugging, aiming for max +/-16K as debug values are 16 bit
 #define D_LPF_RAW_SCALE 25
-#define D_LPF_FILT_SCALE 22
+#define DEBUG_D_PRE_TPA_SCALE 10
```

Comment:
bbe: 
```
$ grep -Iirn D_LPF ./js
./js/flightlog_fields_presenter.js:846:        'D_LPF' : {
./js/flightlog_fields_presenter.js:847:            'debug[all]':'D-Term [D_LPF]',
./js/flightlog_fielddefs.js:337:        "D_LPF",
```
upon skim-read, i do not see any `SCALE` nor static `22` associated.
may need to add translation (`flightlog_parser.js:358`), but unsure :man_shrugging:

Reply:
See https://github.com/betaflight/betaflight/pull/12722/commits/4361d320b43797f84f7938134ba6cd77878a8198

Reply:
You give me mixed messages here. Do we need to update the corresponding debug channel in the blackbox viewer or not?

Reply:
No, this change improves the blackbox scaling so that the D_LPF Debug is easier to interpret.

There are many other debugs which require attention, including the D_LPF debug.  The D_LPF debug from memory requires attention in some of its other channels.

Reply:
Okay, gotcha